src/test/java/com/uid2/client/decrypt_response.py

```diff
@@ -0,0 +1,40 @@
+import base64
```

Comment:
What are these python files included for?

Reply:
there's a comment in PublisherTests about them, but added comments to python:
\# This script was used to generate envelopes used in the PublisherTests java class
and
\# This script was used for the PublisherTests.decryptGenerateResponse() test, to ensure python and java had identical results